Content/Items/Haunted/Accessories.TaintedGreataxe.cs

```diff
@@ -67,7 +68,7 @@ private void ModifyHitNPC(Player player, Item Item, NPC target, ref NPC.HitModif
 			if (!Equipped(player))
 				return;
 
-			TaintedGreataxe item = GetEquippedInstance(player) as TaintedGreataxe;
+			var item = GetEquippedInstance(player) as TaintedGreataxe;
```

Comment:
these onhits probably need to be synced too but I'm not looking at these in this PR